PWGCF/EbyEFluctuations/Tasks/MeanPtFlucIdentified.cxx

```diff
@@ -520,43 +558,64 @@ struct meanPtFlucId {
     hist.fill(HIST("QA/after/innerParam/h2_pvsm2"), track.mass() * track.mass(), track.tpcInnerParam());
   }
 
+  // Moments Calculation:
+  void moments(double pt, double* Q1, double* Q2, double* Q3, double* Q4)
```

Comment:
Have you considered to use references instead of pointers? 
It is usually cleaner, safer and more representative